src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machineArray = new Machine[3];
+        machineArray[0] = new Truck();
+        machineArray[1] = new Bulldozer();
+        machineArray[2] = new Excavator();
 
+        for (Machine el : machineArray) {
```

Comment:
```suggestion
        for (Machine machine : machineArray) {
```
якщо буде розростатись тіло циклу el стане не дуже читабельним